include/envoy/stats/stat_data_allocator.h

```diff
@@ -35,8 +35,8 @@ class StatDataAllocator {
    * @return CounterSharedPtr a counter, or nullptr if allocation failed, in which case
    *     tag_extracted_name and tags are not moved.
    */
-  virtual CounterSharedPtr makeCounter(absl::string_view name, std::string&& tag_extracted_name,
-                                       std::vector<Tag>&& tags) PURE;
+  virtual CounterSharedPtr makeCounter(StatName name, absl::string_view tag_extracted_name,
+                                       const std::vector<Tag>& tags) PURE;
```

Comment:
Did makeCounter() used to modify tags and now it doesn't? Or is the omission of `const` in the old signature a mistake?